### PR TITLE
changed github example to use FSharp.Formatting

### PR DESCRIPTION
**Pull request by cloudRoutine**

### CODE REVIEW

**Comment by forki** [comment]
Thx

File: docs/content/github-dependencies.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ Or if you use ``.`` for the directory, the file will be placed under the root of
 You can also reference a complete [github.com](http://www.github.com) repository by specifying the repository id in the [`paket.dependencies` file](dependencies-file.html):
 
     [lang=paket]
-    github hakimel/reveal.js                                          // master branch
-    github hakimel/reveal.js:2.6.2                                    // version 2.6.2
-    github hakimel/reveal.js:822a9c937c902807e376713760e1f07845951d5a // specific commit 822a9c937c902807e376713760e1f07845951d5a
+    github tpetricek/FSharp.Formatting                                         // master branch 
+    github tpetricek/FSharp.Formatting:2.13.5                                  // version 2.6.2
```
Comment (mexx) [inline docs/content/github-dependencies.md]: can we please adjust the version in the comment also

Comment (forki) [inline docs/content/github-dependencies.md]: Oupsi. Yes please send another pr
